test/Altinn.App.Api.Tests/Altinn.App.Api.Tests.csproj

```diff
@@ -27,33 +27,21 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Altinn.App.Api\Altinn.App.Api.csproj" />
+    <ProjectReference Include="../../src/Altinn.App.Api/Altinn.App.Api.csproj" />
 
-    <!-- <ProjectReference Include="..\..\src\Altinn.App.Api\Altinn.App.Api.csproj">
-      <CopyToOutputDirectory>lib\$(TargetFramework)\*.xml</CopyToOutputDirectory>
+    <!-- <ProjectReference Include="../../src/Altinn.App.Api/Altinn.App.Api.csproj">
+      <CopyToOutputDirectory>lib/$(TargetFramework)/*.xml</CopyToOutputDirectory>
     </ProjectReference> -->
-    <ProjectReference Include="..\Altinn.App.Common.Tests\Altinn.App.Common.Tests.csproj" />
+    <ProjectReference Include="../Altinn.App.Common.Tests/Altinn.App.Common.Tests.csproj" />
```

Comment:
So is `Altinn.App.Common.Tests` a shared library or a project with unit tests?

Reply:
Shared library, re PR description

> Common.Tests is perhaps a little weird naming considering its not holding tests

Are you bringing up the same point? If so, maybe we should rename it `Altinn.App.Tests.Common` or something

Reply:
could maybe put the `Shared` folder in there as well

Reply:
>could maybe put the Shared folder in there as well

Yes exactly

> Are you bringing up the same point? If so, maybe we should rename it Altinn.App.Tests.Common or something

Yes, I somehow got confused reading the diff that the tests had not been moved.

Reply:
Renamed the project and moved the shared files into it